app/controllers/api/v2/hosts_controller.rb

```diff
@@ -262,6 +262,22 @@ def power
         end
       end
 
+      api :GET, '/hosts/:id/power', N_('Fetch the status of whether the host is powered on or not. Supported hosts are vm, bmc or virt.')
```

Comment:
VM and Virt is the same, isn't it?

Reply:
Ok before I change this again and its not correct. What should be here exactly ?

Shall I just remove the `Supports . . . ` part or ? or `VM's and BMC's` ?

Reply:
Something like `Supported hosts are VMs and physical hosts with BMCs.` - or just omit it.